plugins/org.eclipse.sirius.doc/doc/user/diagrams/Diagrams.textile

```diff
@@ -853,7 +853,11 @@ It is also possible to pin/unpin diagram elements thanks to the palette. Multipl
 Notes:
 
 * Pin/unpin information are persistent and saved in __.aird__ session file.
-* it is not possible to pin/unpin edges or compartments.
+* It is not possible to pin/unpin compartments.
+* For edges, the default layout behavior can be different according to the source/target pinned status:
+** The edge, the source and the target are pinned: nothing moves during arrange all.
+** The edge is not pinned and, the source and the target are pinned: the edge is moved to keep only a minimum number of bendpoints.
```

Comment:
Maybe "the edge is arranged" instead of "the edge is moved" here.

Reply:
Done